readthedocs/proxito/views/hosting.py

```diff
@@ -115,6 +115,23 @@ def get(self, request):
                         "filepath": "/docs/index.rst",
                     },
                 },
+                "search": {
+                    "project": project.slug,
+                    "version": version.slug,
+                    "api_endpoint": "/_/api/v3/search/",
+                    # TODO: figure it out where this data comes from
+                    "filters": [
+                        [
+                            "Search only in this project",
+                            f"project:{project.slug}/{version.slug}",
+                        ],
+                        [
+                            "Search only on subprojects",
```

Comment:
This includes the current project and it's subprojects.


```suggestion
                            "Search subprojects",
```

But this filter should probably be done dynamically, since it only makes sense in superprojects.

Reply:
> But this filter should probably be done dynamically, since it only makes sense in superprojects.

Yes. This data will come from the users' settings somehow, once we define the user's API in https://github.com/readthedocs/readthedocs-client/issues/13 -- BTW, if you have ideas about this, please comment there 👍🏼 

For now, I'm just returning some default data here to be able to move forward with the implementation and the tests.